docs/source/installation/gitops.md

```diff
@@ -156,7 +156,7 @@ kubectl wait --for='condition=Reconciled' --timeout=10m nodeconfigs.scylla.scyll
 :::{code-block} shell
 :substitutions:
 
-kubectl -n=local-csi-driver apply --server-side -f=https://raw.githubusercontent.com/{{repository}}/{{revision}}/examples/common/local-volume-provisioner/local-csi-driver/{00_namespace,00_scylladb-local-xfs.storageclass,10_csidriver,10_driver.serviceaccount,10_provisioner_clusterrole,20_provisioner_clusterrolebinding,50_daemonset}.yaml
+kubectl -n=local-csi-driver apply --server-side -f=https://raw.githubusercontent.com/{{repository}}/{{revision}}/examples/common/local-volume-provisioner/local-csi-driver/{00_clusterrole_def,00_clusterrole,00_namespace,00_scylladb-local-xfs.storageclass,10_csidriver,10_serviceaccount,20_clusterrolebinding,50_daemonset}.yaml
```

Comment:
`00_clusterrole_def_openshift.yaml` is missing

Reply:
Yeah, true. I omitted it assuming this doesn't fit to the generic install instructions (`00_clusterrole_def_openshift.yaml` adds rules for `securitycontextconstraints` specific only to OpenShift). How about splitting this into "Generic" and "OpenShift" tabs? [PTAL](https://github.com/scylladb/scylla-operator/compare/474d442ec7e1603c597b7ef695e60abf20264f4f..e377209caf0090313b55c0675c3323248b406552)

Reply:
While it's not a bad idea per se, I don't think the distinction is necessary, we already advise deploying the openshift-specific roles in other places, see e.g. https://operator.docs.scylladb.com/stable/installation/gitops.html#scylla-operator and https://github.com/scylladb/scylla-operator/blob/b15e71b5938537c5e727dfed19bc5b5f2375ccf3/deploy/operator/00_operator.clusterrole_def_openshift.yaml (aggregated to https://github.com/scylladb/scylla-operator/blob/b15e71b5938537c5e727dfed19bc5b5f2375ccf3/deploy/operator.yaml).
Given we don't broadcast our support for openshift, it might only introduce more confusion, so I'd just keep a single generic command with the openshift-specific files included.

Reply:
Sure, thanks for the explanation. Let's be consistent.